maintainers/scripts/update.nix

```diff
@@ -120,7 +120,7 @@ let
       if pathContent == null then
         builtins.throw "Attribute path `${path}` does not exist."
       else
-        packagesWithPath prefix (path: pkg: builtins.hasAttr "updateScript" pkg)
+        packagesWithPath prefix (path: pkg: !(builtins.hasAttr "updateScript" pkg && pkg.updateScript == null))
```

Comment:
not blocking, but wonder why we didn't just use `pkg ? updateScript` instead of the `hasAttr` bit